src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
+import { CommentInfo } from '../CommentInfo';
+import { Comments } from '../../types/Comments';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  comments: Comments[];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <ul className="CommentList">
+    {comments.map(comment => (
+      <li key={comment.id}>
+        <CommentInfo comment={comment} />
+      </li>
+    ))}
+  </ul>
```

Comment:
I think you should remove bullets from this list in CSS file, as for me, it doesn't look good.